libs/sdk-ui-kit/src/EditableLabel/EditableLabel.tsx

```diff
@@ -0,0 +1,271 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { Component, ReactNode, RefObject } from "react";
+import bindAll from "lodash/bindAll";
+import uniqueId from "lodash/uniqueId";
+import identity from "lodash/identity";
+import TextareaAutosize from "react-textarea-autosize";
+import cx from "classnames";
+
+import { Overlay } from "../Overlay";
+import { ENUM_KEY_CODE } from "../typings/utilities";
+
+import { IEditableLabelProps, IEditableLabelState } from "./typings";
+
+/**
+ * @internal
+ */
+export class EditableLabel extends Component<IEditableLabelProps, IEditableLabelState> {
+    static defaultProps = {
+        children: false,
+        className: "",
+        maxLength: 100000,
+        maxRows: 1,
+        onCancel: identity,
+        onEditingStart: identity,
+        onChange: identity,
+        placeholder: "",
+        scrollToEndOnEditingStart: true,
+        textareaInOverlay: false,
+        autofocus: false,
+    };
+    private readonly root: RefObject<any>;
+    private readonly textarea: RefObject<HTMLTextAreaElement>;
+
+    constructor(props: IEditableLabelProps) {
+        super(props);
+
+        bindAll(this, ["edit", "selectAndFocus", "onKeyDown", "onSubmit", "onChange", "onDocumentClick"]);
```

Comment:
Can we avoid the need for `bindAll` with arrow members?

Reply:
Fixed.